### ISSUE TITLE
(1.17) Concurrent Modification Exception

### ISSUE THREAD

**Issue by IBeHunting** [Nathat23/StackMob-5#225]
Hello!

I'm getting an error related to a certain event on the newest version of StackMob (5.6.1)
```
21.06 05:20:39 [Server] WARN [StackMob] Task #418 for StackMob v5.6.1 generated an exception
21.06 05:20:39 [Server] INFO java.util.ConcurrentModificationException: null
21.06 05:20:39 [Server] INFO at java.util.HashMap$HashIterator.nextNode(HashMap.java:1584) ~[?:?]
21.06 05:20:39 [Server] INFO at java.util.HashMap$ValueIterator.next(HashMap.java:1612) ~[?:?]
21.06 05:20:39 [Server] INFO at uk.antiperson.stackmob.tasks.MergeTask.run(MergeTask.java:26) ~[?:?]
21.06 05:20:39 [Server] INFO at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftTask.run(CraftTask.java:100) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1488) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:476) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1404) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1180) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[patched_1.17.jar:git-Paper-"bc4b23e"]
21.06 05:20:39 [Server] INFO at java.lang.Thread.run(Thread.java:831) [?:?]
```

Besides that error showing up in console, the plugin seems to work just fine. Keep up the good work!

**Comment by Nathat23** [comment]
Are you able to run a test build?
https://cdn.discordapp.com/attachments/513322457101238291/856578395865088010/StackMob-5.6.1-test.jar
This won't fix the issue but should help find the problem?

**Comment by Nathat23** [comment]
Fixed in 539836e60f5db5106623a45bd081ac542d5e7649, update has been released on spigot.
